pkg/image/importer/importer.go

```diff
@@ -397,6 +418,7 @@ func importRepositoryFromDocker(ctx gocontext.Context, retriever RepositoryRetri
 			importDigest.Err = err
 			continue
 		}
+		importDigest.Err = calculateImageSize(ctx, repo, importDigest.Image)
```

Comment:
>  it's unnecessarily calculates the size. 

@soltysh @smarterclayton  I added the check to calculate the size only when it is equal to zero. This would eliminate unnecessary requests if we have all the information.